package.json

```diff
@@ -1,12 +1,30 @@
 {
   "private": true,
   "dependencies": {},
+  "scripts": {
+    "lint": "eslint --ext js,jsx,ts,tsx .",
+    "format": "prettier --write \"**/*.{ts,tsx,js,jsx,json}\""
+  },
+  "lint-staged": {
+    "*.{ts,js,tsx,jsx}": [
+      "eslint --fix",
+      "prettier --write"
+    ],
+    "*.json": [
+      "prettier --write"
+    ],
+    "messages/*.json": [
+      "yarn lint:locales"
+    ]
+  },
   "devDependencies": {
     "@types/node": "^12.7.2",
-    "eslint": "^6.2.2",
-    "eslint-config-vtex": "^11.0.0",
-    "eslint-config-vtex-react": "^5.0.1",
-    "prettier": "^1.18.2",
-    "typescript": "^3.5.3"
+    "@vtex/prettier-config": "^0.3.6",
+    "eslint": "^7.14.0",
+    "eslint-config-vtex": "^12.8.11",
```

Comment:
Bump to our latest version: [14.x](https://github.com/vtex/typescript/blob/main/packages/eslint-config-vtex/package.json)